package_builder/build_serialbox.sh

```diff
@@ -28,40 +30,55 @@ if [[ -z ${install_dir} ]]; then
     exitError 3225 ${LINENO} "package install dir has to be specified"
 fi
 
+# Setup
 echo $@
 base_path=$PWD
 setupDefaults
 
 if [[ ${install_local} == "yes" ]]; then
-    install_args="--local"
+    install_path_prefix_="${base_path}/install"
 else
-    install_args="-i ${install_dir}/serialbox/${c_}/"
+    install_path_prefix_="${install_dir}/serialbox"
 fi
 
-for c_ in ${compilers[@]}; do
```

Comment:
were there problems with these previous version where we loop over the compilers? Why that couldnt work?
Because what if we want build only for 2 out of all 4 compilers?